dependencies.props

```diff
@@ -1,18 +1,18 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
-    <CoreFxCurrentRef>c9fa785e7049638603fca518d56198f60d901203</CoreFxCurrentRef>
-    <CoreClrCurrentRef>fcd99ed86a703a16698fcd0027707739c8ef3501</CoreClrCurrentRef>
+    <CoreFxCurrentRef>38ea755f8ef48c177c78ccc52b3df1f30c08db3d</CoreFxCurrentRef>
+    <CoreClrCurrentRef>38ea755f8ef48c177c78ccc52b3df1f30c08db3d</CoreClrCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-01</CoreFxExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview1-25320-02</CoreFxExpectedPrerelease>
   </PropertyGroup>
 
   <!-- Full package version strings that are used in other parts of the build. -->
   <PropertyGroup>
-    <CoreClrPackageVersion>2.0.0-preview2-25316-03</CoreClrPackageVersion>
+    <CoreClrPackageVersion>2.1.0-preview1-25319-05</CoreClrPackageVersion>
```

Comment:
This is the same situation that we have in Core-Setup (https://github.com/dotnet/core-setup/pull/2453), but in CoreCLR the fix can be made now because all CoreCLR dependencies have a new release/2.0.0 build out. https://github.com/dotnet/core-setup/pull/2455 is the Core-Setup fix, but it's waiting for a WCF 2.0.0 build. /cc @weshaggard